### ISSUE TITLE
"Please upgrade this package",  error when attempting to connect Colab to local runtime

### ISSUE THREAD

**Issue by iampatgrady** [ikeyasu/docker-colab-local#1]
Steps to reproduce:

1) $ docker pull ikeyasu/colab-local
2) $ docker run -it --rm -p 8081:8081 --cap-add SYS_ADMIN --device /dev/fuse --security-opt apparmor=unconfined ikeyasu/colab-local:latest
3) connect colab 8081 local

ERROR:
[E 15:48:59.048 NotebookApp] Rejecting connection: Requested version (0.0.1a2) > Current version (0.0.1a1). Please upgrade this package.

**Comment by aYasukiIkeuchi** [comment]
Sorry for late response. I'll check soon.

**Comment by ikeyasu** [comment]
sorry for late response. I'll check soon.

**Comment by RKelln** [comment]
I'm seeing this as well, the troubleshooting link from Google says:

> Ensure the jupyter_http_over_ws Jupyter extension is enabled and up to date on your machine.
```
 pip install --upgrade jupyter_http_over_ws>=0.0.1a3 && \
  jupyter serverextension enable --py jupyter_http_over_ws
```
Likely the [RUN command in the Dockerfile](https://github.com/ikeyasu/docker-colab-local/blob/cda0b3b6d5fd17d57945312b6f170902b23da7a6/Dockerfile#L182) can be updated and it should work. I can try testing that soon.

EDIT: Tested and that allows for colab to connect. I ran into a bunch of other issues getting a docker image to build, so I'm wondering if you'd be interested in taking a look and trying to build a refreshed image?
